apps/backend/src/config/config.service.ts

```diff
@@ -2,6 +2,17 @@ import {SequelizeOptions} from 'sequelize-typescript';
 import AppConfig from '../../config/app_config';
 import {StartupSettingsDto} from './dto/startup-settings.dto';
 
+export const sensitiveKeys = [
+  /cookie/i,
+  /passw(or)?d/i,
+  /^pw$/,
+  /^pass$/i,
+  /secret/i,
+  /token/i,
+  /api[-._]?key/i,
+  /data/i
+];
+
```

Comment:
Can this be a function as part of the module instead of exported separately.

Reply:
I have moved it into ConfigService assuming that's what you meant here.